src/SldStyleParser.ts

```diff
@@ -2181,6 +2190,75 @@ export class SldStyleParser implements StyleParser<string> {
     }];
   }
 
+  /**
+   * Translates an anchor-setting into SLD-anchor-numbers
+   */
+  getSldAnchorPointFromAnchor(anchor: TextSymbolizerAnchor, dimension: 'x' | 'y'): number {
+    if (!anchor || isGeoStylerFunction(anchor)) {
+      return 0;
+    }
+    // As explained in https://docs.geoserver.org/main/en/user/styling/sld/reference/labeling.html#anchorpoint,
+    // we have the following translation for anchors:
+    // x-dimension
+    //   left -> 0.0
+    //   center -> 0.5
+    //   right -> 1.0
+    // y-dimension
+    //   top -> 1.0
+    //   middle -> 0.5
+    //   bottom -> 0.0
+
+    if (dimension==='x') {
```

Comment:
It would be great, if you could add spaces around the conditions, e.g.
```
    if (dimension === 'x') {
```
instead of
```
     if (dimension==='x') {
```

(applies for all if-statements in this function).